src/Connector/RedmineConnector.php

```diff
@@ -206,12 +206,25 @@ public function assigned($user = 'me') {
         if (!isset($tickets[$project])) {
           $tickets[$project] = [];
         }
-        $tickets[(string) $node->project['name']][(string) $node->id] = (string) $node->subject;
+        $tickets[(string) $node->project['name']][(string) $node->id] = [
```

Comment:
This is the assigned function? I couldn't see it an initial look and those two seem to work.